normalize.css

```diff
@@ -0,0 +1,509 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
+
+html {
+<<<<<<< HEAD
+  line-height: 1.15; /* 1 */
+  -webkit-text-size-adjust: 100%; /* 2 */
+=======
+    line-height: 1.15; /* 1 */
+    -webkit-text-size-adjust: 100%; /* 2 */
+>>>>>>> 38bc1606d417339eafef6982d3fa4542502dfc23
+}
```

Comment:
It seems that this file has been merged conflicted! No idea how as I saw holly fix the conflicts, but never mind! 

Simple solution is to just replace the contents of this file with a fresh copy of normalize